vllm/platforms/cpu.py

```diff
@@ -186,16 +184,14 @@ def check_and_update_config(cls, vllm_config: VllmConfig) -> None:
 
         cache_config = vllm_config.cache_config
 
-        ipex_available = find_spec("intel_extension_for_pytorch") is not None
+        if cache_config.block_size is None:
+            cache_config.block_size = 128
 
-        if cache_config and cache_config.block_size is None:
-            cache_config.block_size = 128 if ipex_available else 16
-
-        if not ipex_available and cache_config.block_size != 16:
-            raise RuntimeError(
-                f"--block-size={cache_config.block_size} requires"
-                " intel_extension_for_pytorch"
-            )
+        if cache_config.block_size % 32 != 0:
+            block_size = cache_config.block_size
```

Comment:
I'm in favor of just erroring out in this case saying that block_size needs to be divisible by 32 (instead of setting the block_size to a value that the user didn't choose)

Reply:
Yes...But my concern is a lot of test cases use 16 by default and I don't want to add more if-else in different files so just round it here.

Reply:
Oh, that's a good point. I agree with you.